fdbserver/fdbserver.actor.cpp

```diff
@@ -471,152 +474,192 @@ static void printHelpTeaser( const char *name ) {
 	fprintf(stderr, "Try `%s --help' for more information.\n", name);
 }
 
+static void printOptionUsage(std::string option, std::string description) {
+	static const std::string OPTION_INDENT("  ");
+	static const std::string DESCRIPTION_INDENT("                ");
+	static const int WIDTH = 80;
+
+	boost::algorithm::trim(option);
+	boost::algorithm::trim(description);
+
+	std::string result = OPTION_INDENT + option + "\n";
+
+	std::stringstream sstream(description);
+	if (sstream.eof()) {
+		printf(result.c_str());
+		return;
+	}
+
+	std::string currWord;
+	sstream >> currWord;
+
+	std::string currLine(DESCRIPTION_INDENT + ' ' + currWord);
+	int currLength = DESCRIPTION_INDENT.size();
+
+	while(!sstream.eof()) {
+		sstream >> currWord;
+
+		if (currLength + static_cast<int>(currWord.size()) + 1 > WIDTH) {
+			result += currLine + '\n';
+			currLine = DESCRIPTION_INDENT + ' ' + currWord;
+		} else {
+			currLine += ' ' + currWord;
+		}
+		currLength = currLine.size();
+	}
+	result += currLine + '\n';
+
+	printf(result.c_str());
+}
+
 static void printUsage( const char *name, bool devhelp ) {
 	printf("FoundationDB " FDB_VT_PACKAGE_NAME " (v" FDB_VT_VERSION ")\n");
 	printf("Usage: %s -p ADDRESS [OPTIONS]\n\n", name);
-	printf("  -p ADDRESS, --public_address ADDRESS\n"
+	printOptionUsage("  -p ADDRESS, --public_address ADDRESS",
 		   "                 Public address, specified as `IP_ADDRESS:PORT' or `auto:PORT'.\n");
-	printf("  -l ADDRESS, --listen_address ADDRESS\n"
-		   "                 Listen address, specified as `IP_ADDRESS:PORT' (defaults to\n");
-	printf("                 public address).\n");
-	printf("  -C CONNFILE, --cluster_file CONNFILE\n"
+	printOptionUsage("  -l ADDRESS, --listen_address ADDRESS\n",
```

Comment:
I know that these won't have any effect on the output, but it might be misleading to readers of the code to have the `\n` scattered throughout. What do you think about removing them all? We can probably also remove all of the leading space within the strings, because that also has no effect on the output.

Reply:
Cleaned up